test/TestExpectations.json

```diff
@@ -1437,10 +1444,10 @@
   },
   {
     "testIdPattern": "[elementhandle.spec] ElementHandle specs ElementHandle.clickablePoint should work for iframes",
-    "platforms": ["linux", "win32"],
+    "platforms": ["linux", "win32", "win32"],
```

Comment:
@OrKoN is this duple ok?

Reply:
didn't notice it, I guess it is fine since our tooling converts it into a set, but we fixed the test and removed the expectation later